### ISSUE TITLE
Sort selected methods by alphabet.

### ISSUE THREAD

**Issue by DasOhmoff** [codecadwallader/codemaid#837]
Hello, thanks for your support and this plugin.

Sometimes one wants to sort some functions in his code. For example in my test classes I would like to sort only my test functions alphabetically, it would be great to have such a feature.

My idea would be to put it into the CodeMaid Spade:
![image](https://user-images.githubusercontent.com/16063186/131489265-37a5c7cb-4c6e-46d3-b79b-3cb7341ee391.png)

**Comment by codecadwallader** [comment]
Thanks for the suggestion!  That makes sense and I like the idea of using Spade to select which items since it has that capability already.

It could be confusing what will happen if you select items that are not next to each other in the list.  Would they all move up in the file to the first member?

**Comment by DasOhmoff** [comment]
> It could be confusing what will happen if you select items that are not next to each other in the list. Would they all move up in the file to the first member?

Yeah, that seems to make sense. Alternatively this sort alphabetically context menu could also not show up if there is a gab between the methods, as I don't think people would use it like that.

**Comment by DasOhmoff** [comment]
Sorry, closed accidentally. Reopened.

**Comment by DasOhmoff** [comment]
Hey, is there any progress on this?

**Comment by codecadwallader** [comment]
No, it does not look like it.
